python/ray/tests/test_actor_failures.py

```diff
@@ -441,7 +441,8 @@ def increase(self):
 @pytest.mark.parametrize(
     "ray_start_cluster_head", [
         generate_internal_config_map(
-            initial_reconstruction_timeout_milliseconds=1000)
+            initial_reconstruction_timeout_milliseconds=1000,
+            num_heartbeats_timeout=10)
```

Comment:
Did you mean to add this?

Reply:
Yes, it was initially passed to the worker nodes